android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/screens/surveyquestions/SurveyQuestionsScreen.kt

```diff
@@ -154,24 +177,43 @@ private fun SurveyQuestionsContent(questions: List<QuestionContentUiModel>) {
 
     ConstraintLayout(modifier = Modifier.fillMaxSize()) {
         val (nextButton) = createRefs()
+        val isLastPage = pagerState.currentPage == pagerState.pageCount - 1
         Button(
-            onClick = {},
+            onClick = {
+                coroutineScope.launch {
+                    pagerState.animateScrollToPage(pagerState.currentPage + 1)
+                }
+            },
             colors = ButtonDefaults.buttonColors(
-                backgroundColor = Color.Transparent
+                backgroundColor = if (isLastPage) Color.White else Color.Transparent
             ),
             elevation = null,
             contentPadding = PaddingValues(0.dp),
+            shape = if (isLastPage) RoundedCornerShape(10.dp) else MaterialTheme.shapes.small,
             modifier = Modifier.constrainAs(nextButton) {
                 bottom.linkTo(parent.bottom, margin = 54.dp)
                 end.linkTo(parent.end, margin = 20.dp)
             }
         ) {
-            Image(
-                painter = painterResource(id = R.drawable.ic_survey_detail_arrow),
-                contentDescription = null,
-                contentScale = ContentScale.FillWidth,
-                modifier = Modifier.size(56.dp),
-            )
+            if (isLastPage) {
+                Text(
+                    text = stringResource(id = R.string.submit_button),
+                    color = Color.Black,
+                    style = Typography.subtitle2,
+                    textAlign = TextAlign.Center,
+                    modifier = Modifier
+                        .width(120.dp)
+                        .height(56.dp)
+                        .wrapContentHeight()
+                )
+            } else {
+                Image(
+                    painter = painterResource(id = R.drawable.ic_survey_detail_arrow),
+                    contentDescription = null,
+                    contentScale = ContentScale.FillWidth,
+                    modifier = Modifier.size(56.dp)
```

Comment:
same, common dimension now

Reply:
Fixed in [`c94847b` (#115)](https://github.com/suho/kmm-ic/pull/115/commits/c94847b78270d2d77ad2d9420e5d518d36b2a874)